fifoqueue/api.w

```diff
@@ -0,0 +1,14 @@
+bring cloud;
+
+pub struct FifoQueueProps extends cloud.QueueProps{}
+
+pub struct PushOptions {
+  groupId: str;
+}
```

Comment:
I think we should also support a deduplicationId.

Reply:
this is default AWS impl